lib/scraper.rb

```diff
@@ -0,0 +1,49 @@
+require 'nokogiri'
+
+require 'open-uri'
+
+require_relative './scraper_instr'
+
+class Scraper < ScraperPlay
+  attr_reader :url
+
+  def initialize(url)
+    super
+
+    @url = url
+  end
+
+  def fetch_title
+    title_cat = []
+
+    first = full_page.css('div#courses .col h3').text
+
+    second = full_page.css('section#youtube .fa-youtube + h3').text
+
+    title_cat << first
+
+    title_cat << second
+
+    title_cat
+  end
+
+  def course1
+    full_page.css('.container-fluid div.row .col-md-4 .bg-black span.font-alt').text
+  end
+
+  def course2
+    full_page.css('section#youtube .container .col-md-6 .bg-white span').text.gsub("\n", '')
```

Comment:
This methods are returning a single string and it is very difficult to distinguish between the different titles. Please use enumerators to have an object with array of strings.

Reply:
I have solve the loop issues when a user choose 'NO'. I have also fixed the rspec testing units to act as supposed.

Reply:
- Hi @promise-J, I have run the cloned repo of your project. I have attached a screenshot to show you how the console output is displayed. 
![Screenshot from 2021-03-26 01-59-26](https://user-images.githubusercontent.com/64894927/112543444-d0bcb080-8dc6-11eb-8d95-e695a7c8b9ee.png)
- Very sorry for the spec comment. There was a bad network when I run Rspec locally and it gave me errors. But the first comment needs to be addressed.